docs/main/concepts/data-engines/data-engines.md

```diff
@@ -154,30 +154,30 @@ Below table identifies few differences among the different OpenEBS Replicated en
 | Suitable for high capacity (>50GB) workloads |  No     |   Yes    | Yes        |
 | Near disk performance                        |  No     |   No     | Yes        |
 
-### Ideal Conditions for selecting OpenEBS Local Engine
+### Ideal Conditions for selecting OpenEBS Local Storage
 
 - When applications are managing replication and availability themselves and there is no need of replication at storage layer. In most such situations, the applications are deployed as `statefulset`.
-- Local Engine is recommended when dedicated local disks are not available for a given application or dedicated storage is not needed for a given application.
+- Local Storage is recommended when dedicated local disks are not available for a given application or dedicated storage is not needed for a given application.
 - When near disk performance is a need along with features like snapshots, volume expansion, pooling of storage from multiple storage devices. 
 
-### Ideal Conditions for selecting OpenEBS Replicated Engine
+### Ideal Conditions for selecting OpenEBS Replicated Storage
 
 - When you need high performance storage using NVMe SSDs and the cluster is capable of NVMeoF. 
 - When you need replication or availability features to protected against node failures.  
-- Replicated Engine is designed for the next gen compute and storage technology and is under active development. 
+- Replicated Storage is designed for the next gen compute and storage technology and is under active development. 
 
-As Replicated Engine is under active development, you can also influence the features that are being built by joining [OpenEBS community on Kubernetes Slack](https://kubernetes.slack.com). If you are already signed up, head to our discussions at [#openebs](https://kubernetes.slack.com/messages/openebs/) channel. 
+As Replicated Storage is under active development, you can also influence the features that are being built by joining [OpenEBS community on Kubernetes Slack](https://kubernetes.slack.com). If you are already signed up, head to our discussions at [#openebs](https://kubernetes.slack.com/messages/openebs/) channel. 
```

Comment:
Need to re-phrase this
For latest updates and discussions, community users are invited to ...